### ISSUE TITLE
Instruction documentations on How to use MarktplaatsUserscript

### ISSUE THREAD

**Issue by 0xMH** [ArnoldDeRuiter/MarktplaatsUserscript#13]
Thanks for your nice project. It'll be helpful if we added a section to the readme explaining how to use this script.

**Comment by ArnoldDeRuiter** [comment]
I agree, I could elaborate more on certain features, like the "Sneltoetsen", or the key combo's, and blacklisting of sellers by name, which are only actually mentioned in the script it self.

Also a how to install section could be a nice addition.

Did you have anything else in mind? Than I can pick it up in one go.

Thanks for your interest in this Userscript. :slightly_smiling_face:
